subprojects/architecture-test/src/changes/archunit_store/dependency-injection-service-scope.txt

```diff
@@ -0,0 +1,96 @@
+Class <org.gradle.api.publish.maven.internal.dependencies.VersionRangeMapper> is not annotated with @ServiceScope in (VersionRangeMapper.java:0)
+Class <org.gradle.api.publish.maven.internal.publisher.MavenPublishers> is not annotated with @ServiceScope in (MavenPublishers.java:0)
+Class <org.gradle.api.reporting.components.internal.TypeAwareBinaryRenderer> is not annotated with @ServiceScope in (TypeAwareBinaryRenderer.java:0)
+Class <org.gradle.api.services.BuildServiceParameters> is not annotated with @ServiceScope in (BuildServiceParameters.java:0)
+Class <org.gradle.api.services.BuildServiceRegistry> is not annotated with @ServiceScope in (BuildServiceRegistry.java:0)
+Class <org.gradle.api.tasks.TaskContainer> is not annotated with @ServiceScope in (TaskContainer.java:0)
+Class <org.gradle.api.testing.toolchains.internal.JvmTestToolchainParameters> is not annotated with @ServiceScope in (JvmTestToolchainParameters.java:0)
+Class <org.gradle.caching.configuration.BuildCacheConfiguration> is not annotated with @ServiceScope in (BuildCacheConfiguration.java:0)
+Class <org.gradle.configuration.CompileOperationFactory> is not annotated with @ServiceScope in (CompileOperationFactory.java:0)
+Class <org.gradle.configuration.ConfigurationTargetIdentifier> is not annotated with @ServiceScope in (ConfigurationTargetIdentifier.java:0)
+Class <org.gradle.configuration.ScriptPluginFactory> is not annotated with @ServiceScope in (ScriptPluginFactory.java:0)
+Class <org.gradle.configuration.internal.ListenerBuildOperationDecorator> is not annotated with @ServiceScope in (ListenerBuildOperationDecorator.java:0)
+Class <org.gradle.configuration.project.ProjectConfigurationActionContainer> is not annotated with @ServiceScope in (ProjectConfigurationActionContainer.java:0)
+Class <org.gradle.configuration.project.ProjectEvaluator> is not annotated with @ServiceScope in (ProjectEvaluator.java:0)
+Class <org.gradle.deployment.internal.DeploymentRegistry> is not annotated with @ServiceScope in (DeploymentRegistry.java:0)
+Class <org.gradle.execution.selection.BuildTaskSelector$BuildSpecificSelector> is not annotated with @ServiceScope in (BuildTaskSelector.java:0)
+Class <org.gradle.execution.taskgraph.TaskExecutionGraphInternal> is not annotated with @ServiceScope in (TaskExecutionGraphInternal.java:0)
+Class <org.gradle.groovy.scripts.internal.ScriptCompilationHandler> is not annotated with @ServiceScope in (ScriptCompilationHandler.java:0)
+Class <org.gradle.initialization.BuildClientMetaData> is not annotated with @ServiceScope in (BuildClientMetaData.java:0)
+Class <org.gradle.initialization.DependenciesAccessors> is not annotated with @ServiceScope in (DependenciesAccessors.java:0)
+Class <org.gradle.initialization.ProjectDescriptorRegistry> is not annotated with @ServiceScope in (ProjectDescriptorRegistry.java:0)
+Class <org.gradle.internal.Factory> is not annotated with @ServiceScope in (Factory.java:0)
```

Comment:
❌ There are quite a few entries in here that don't make any sense. For example, `Factory` should never have a service scope. Can you add the corresponding exceptions in the code so the freeze file becomes smaller?